scripts/twister

```diff
@@ -464,6 +464,15 @@ Artificially long but functional example:
         action="store",
         help="Load list of tests and platforms to be run from file.")
 
+    parser.add_argument(
+        "-q",
```

Comment:
I guess it's better not to use '-q' for this purpose as usually '-q' stands for 'quiet'. So it's quite misleading in this case.

Reply:
I removed `-q`, only full `--quarantine-list` will work now